brian2/sphinxext/generate_examples.py

```diff
@@ -139,8 +139,8 @@ def main(rootpath, destdir):
         output += '\n'.join(['    ' + line for line in afterdoccode.split('\n')])
         output += '\n\n'
 
-        eximgpattern = os.path.join(eximgpath, '%s.*.png' % exname)
-        images = glob.glob(eximgpattern)
+        eximgpattern = os.path.join(eximgpath, '%s.*' % exname)
```

Comment:
Let's make this an f-string!

Reply:
It's weird that it wasn't an f-string already, actually... I changed the remaining `%` strings in the file as well.